packages/ui/hooks/fuse/useExtraPoolInfo.ts

```diff
@@ -8,17 +8,16 @@ export const useExtraPoolInfo = (comptrollerAddress: string) => {
   const { data } = useQuery(['ExtraPoolInfo', currentChain.id, comptrollerAddress], async () => {
     if (comptrollerAddress) {
       const comptroller = fuse.createComptroller(comptrollerAddress);
+      const oracle = fuse.getPriceOracle(await comptroller.callStatic.oracle());
```

Comment:
I think we could have left it in the Promise array, so everything is fetched simultaneously.